File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseThirdparty.java
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import app.editors.manager.mvp.models.base.Base;
import app.editors.manager.mvp.models.user.Thirdparty;

public class ResponseThirdparty extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private List<Thirdparty> response;

    public List<Thirdparty> getResponse() {
        return response;
    }

    public void setResponse(List<Thirdparty> response) {
        this.response = response;
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseThirdparty.kt
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import app.documents.core.network.common.models.BaseResponse;
import app.documents.core.network.manager.models.user.Thirdparty;

public class ResponseThirdparty extends BaseResponse {

    @SerializedName(BaseResponse.KEY_RESPONSE)
    @Expose
    private List<Thirdparty> response;

    public List<Thirdparty> getResponse() {
        return response;
    }

    public void setResponse(List<Thirdparty> response) {
        this.response = response;
    }
}
